library/src/test/java/com/datatorrent/lib/formatter/JsonFormatterTest.java

```diff
@@ -35,6 +35,8 @@
 
 import org.apache.commons.io.FileUtils;
 
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.collect.Lists;
 
```

Comment:
Please update the test cases here to include additional assertions based on the autometric counters added.